compiler/src/iree/compiler/Codegen/LLVMGPU/test/ROCDL/pipeline_vector_distribute_gfx942.mlir

```diff
@@ -1312,3 +1314,127 @@ module {
 // MEMORY-LABEL: func.func @attention_gather_k
 // MEMORY-COUNT-3: memref.alloc
 // MEMORY-NOT:     memref.alloc
+
+// -----
+
+#translation = #iree_codegen.translation_info<pipeline = LLVMGPUVectorDistribute workgroup_size = [256, 1, 1] subgroup_size = 64>
+#lowering_config = #iree_gpu.lowering_config<{reduction = [0, 0, 0, 0, 0, 512], workgroup = [1, 1, 1, 32, 0, 0]}>
+
+// {indexing_maps = [
+//   affine_map<(d0, d1, d2, d3, d4) -> (d0, d1, d2, d3)>,
+//   affine_map<(d0, d1, d2, d3, d4) -> (d0, d1, d4, d3)>,
+//   affine_map<(d0, d1, d2, d3, d4) -> (d0, d1, d2, d4)
+// ],
+// iterator_types = ["parallel", "parallel", "parallel", "reduction", "parallel"]
+// }
+#qk_config = {
+  lowering_config = #iree_gpu.lowering_config<{
+    subgroup_basis = [[1, 1, 1, 1, 1], [0, 1, 2, 3, 4]],
+    thread_basis = [[1, 1, 1, 1, 64], [0, 1, 2, 3, 4]],
+    thread = [0, 0, 0, 8, 8]
+  }>
+}
+
+//{indexing_maps = [
+//     affine_map<(d0, d1, d2, d3, d4) -> (d0, d1, d2, d4)>,
+//     affine_map<(d0, d1, d2, d3, d4) -> (d0, d1, d3, d4)>,
+//     affine_map<(d0, d1, d2, d3, d4) -> (d0, d1, d2, d3)>
+// ],
+// iterator_types = ["parallel", "parallel", "parallel", "parallel", "reduction"]
+// }
+#pv_config = {
+  lowering_config = #iree_gpu.lowering_config<{
+    subgroup_basis = [[1, 1, 1, 4, 1], [0, 1, 2, 3, 4]],
+    thread_basis = [[1, 1, 1, 1, 64], [0, 1, 2, 3, 4]],
+    thread = [0, 0, 0, 8, 8]
+  }>
+}
+
+module {
+  hal.executable public @decode_attn_dispatch_0 {
```

Comment:
This test should be in pipeline_vector_distribute_gfx942_reduction.mlir (or whever the reduction test file is called)

Reply:
Why? all other attention tests are here...